realm/realm-library/src/main/java/io/realm/RealmConfiguration.java

```diff
@@ -66,7 +66,7 @@
     public static final int KEY_LENGTH = 64;
 
     private static final Object DEFAULT_MODULE;
-    private static final RealmProxyMediator DEFAULT_MODULE_MEDIATOR;
+    static final RealmProxyMediator DEFAULT_MODULE_MEDIATOR;
```

Comment:
It was used by the `schema()` in SyncConfiguration, but that was removed again, so putting back `private`